transcom/mymove
# B 21524 update counselor approved email int

## [B-21524](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1020745)

## Summary

This PR updates the counselor approved email to include more information about storage, pro gear and if a move is an actual expense reimbursement.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test
**Test non actual expense move**

1. Instead of runnning make server_run run this: 
`EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run `
and then run make client_run
2. Create a PPM move
3. Login as a Service counselor and submit the move details
4. Confirm there is no section that reads "Please Note: Your PPM has been designated as Actual Expense Reimbursement."

**Test actual expense move**

1. Create a PPM move
2. in the database set its "isActualExpenseReimbursement" to true
3. Login as a Service counselor and submit the move details
6. Confirm there is a section that reads "Please Note: Your PPM has been designated as Actual Expense Reimbursement." 

### Backend


## Screenshots
**_ActualExpenseReimbursement_**
<img width="823" alt="Screenshot 2024-10-28 at 8 37 38 AM" src="https://github.com/user-attachments/assets/f6c9a820-e70f-48eb-adce-d54af115e53a">

**_Not ActualExpenseReimbursement_**
<img width="780" alt="Screenshot 2024-10-28 at 8 29 04 AM" src="https://github.com/user-attachments/assets/611e7825-4b7b-475e-b807-c8d6cf35cf7c">


**_AC Email:_**
<img width="1041" alt="Screenshot 2024-10-29 at 8 46 38 AM" src="https://github.com/user-attachments/assets/f7289379-b453-41c5-b917-a08a19bc0ef8">

[Email #2 - Your counselor has approved your move details update - 2 Oct 24 (1).docx](https://github.com/user-attachments/files/17557728/Email.2.-.Your.counselor.has.approved.your.move.details.update.-.2.Oct.24.1.docx)
